packages/webpack.config.js

```diff
@@ -18,6 +18,15 @@ const packageJson = require(resolveApp('package.json'));
 console.log(`Building ${packageJson.name}`);
 
 module.exports = env => {
+    const devServer = {
+        port: 3000,
+        host: '0.0.0.0'
+    };
+    if (env.buildTest) {
+        devServer.openPage = 'test';
```

Comment:
What is this part used for?

Reply:
openPage opens the test page if we're building a /test directory.
Or do you mean what is it used for if we aren't?

Reply:
I guess I mean how does it know what test page to open.

Reply:
It assumes there's an index.html. And just opens the test directory.